src/videoCache.js

```diff
@@ -59,13 +59,20 @@ function wrapURI(uri, impUrl) {
  * @param {CacheableBid} bid
  */
 function toStorageRequest(bid) {
+  const conf = config.getConfig();
   const vastValue = bid.vastXml ? bid.vastXml : wrapURI(bid.vastUrl, bid.vastImpUrl);
+
   let payload = {
     type: 'xml',
     value: vastValue,
     ttlseconds: Number(bid.ttl)
   };
 
+  if (conf.cache && conf.cache.url && conf.cache.vasttrack) {
+    payload.bidder = bid.bidder;
+    payload.bidid = bid.adId;
```

Comment:
this should be `bid.requestId` rather than `bid.adId`

Reply:
Already fixed